WMFModel/WMFArticle+CoreDataProperties.h

```diff
@@ -0,0 +1,29 @@
+#import "WMFArticle+CoreDataClass.h"
+
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface WMFArticle (CoreDataProperties)
+
++ (NSFetchRequest<WMFArticle *> *)fetchRequest;
+
+@property (nonatomic) BOOL isBlocked;
```

Comment:
More personal preference than anything… but any reason to change the name to blocked? Blacklisted seems more clear to me than blocked which has a more generic sound… 

Related: now that the model is being changed, should we separate this into 2 properties? one for not using in the feed and one for not using in notifications? So the intent of each is more clear?